src/components/DelegationDashboard/Chat/NewChat.tsx

```
@@ -0,0 +1,166 @@
+							<List.Item
+								key={delegate?.address}
+								onClick={() => handleStartChat(delegate?.address)}
+								className='cursor-pointer border-section-light-container p-0'
+							>
+								<Card
+									key={delegate?.address}
+									className='w-full rounded-none border-t-0'
+									bodyStyle={{ alignItems: 'center', display: 'flex', gap: '0.5rem', width: '100%' }}
+									size='small'
+								>
+									{delegate?.address && delegate?.address?.startsWith('0x') ? (
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)